test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/CodelessEndToEndTests.cs

```diff
@@ -41,28 +40,30 @@ public async Task CodelessFunction_Invokes_HttpTrigger(string path, string worke
                 var provider = new TestCodelessFunctionProvider(metadata, null);
 
                 var functions = allowedList != null ? new[] { allowedList, "testFn" } : null;
-                var host = StartLocalHost(baseTestPath, sourceFunctionApp, functions, new List<IFunctionProvider>() { provider }, testEnvironment);
+                using (var host = StartLocalHost(baseTestPath, sourceFunctionApp, functions, new List<IFunctionProvider>() { provider }, testEnvironment))
```

Comment:
The tests in here weren't disposing of their hosts, which meant they'd run forever in the background. `Validate_HostLogs` validates the "host lock lease" message -- which would never fire because one of these hosts was hanging onto it.